stac_fastapi/api/stac_fastapi/api/app.py

```diff
@@ -410,7 +410,7 @@ def add_health_check(self):
         @mgmt_router.get("/_mgmt/ping")
         async def ping():
             """Liveliness/readiness probe."""
-            return {"message": "PONG"}
+            return {"message": "PONGO"}
```

Comment:
undo this change

Reply:
Sorry, I forgot to change it, I already did it